### PR TITLE
[SPARK-18049][MLLIB][TEST] Add missing tests for truePositiveRate and weightedTruePositiveRate

### PR DESCRIPTION
**Pull request by zhengruifeng**
## What changes were proposed in this pull request?

Add missing tests for `truePositiveRate` and `weightedTruePositiveRate` in `MulticlassMetricsSuite`
## How was this patch tested?

added testing

### CODE REVIEW

File: mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala [hunk 1]
```diff
@@ -73,7 +73,7 @@ class RegressionMetrics @Since("2.0.0") (
 
   /**
    * Returns the variance explained by regression.
-   * explainedVariance = $\sum_i (\hat{y_i} - \bar{y})^2 / n$
+   * explainedVariance = $\sum_i (\hat{y_i} - \bar{y})^2^ / n$
```
Comment (srowen) [inline mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala]: Hm, I was going to say that this shouldn't be how you write exponents in latex, but, yeah it doesn't render correctly whereas `R^2^` below does. OK, if this fixes that issue, good.

File: mllib/src/test/scala/org/apache/spark/mllib/evaluation/RankingMetricsSuite.scala [hunk 2]
```diff
@@ -65,5 +65,4 @@ class RankingMetricsSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(metrics.ndcgAt(1) ~== 0.5 absTol eps)
     assert(metrics.ndcgAt(2) ~== 0.30657 absTol eps)
   }
-
```
Comment (srowen) [inline mllib/src/test/scala/org/apache/spark/mllib/evaluation/RankingMetricsSuite.scala]: I'd just back this out if there's no other change in this file.

File: mllib/src/test/scala/org/apache/spark/mllib/evaluation/MultilabelMetricsSuite.scala [hunk 3]
```diff
@@ -47,7 +47,7 @@ class MultilabelMetricsSuite extends SparkFunSuite with MLlibTestSparkContext {
     val scoreAndLabels: RDD[(Array[Double], Array[Double])] = sc.parallelize(
       Seq((Array(0.0, 1.0), Array(0.0, 2.0)),
         (Array(0.0, 2.0), Array(0.0, 1.0)),
-        (Array(), Array(0.0)),
+        (Array.empty[Double], Array(0.0)),
```
Comment (srowen) [inline mllib/src/test/scala/org/apache/spark/mllib/evaluation/MultilabelMetricsSuite.scala]: Normally would say don't bother with this either but I know you just cleaned up a lot of this for consistency. I'm neutral about whether it's more readable and it's only test code.

File: mllib/src/test/scala/org/apache/spark/mllib/evaluation/RankingMetricsSuite.scala (hunk 2 above)
Comment (zhengruifeng) [inline mllib/src/test/scala/org/apache/spark/mllib/evaluation/RankingMetricsSuite.scala]: ok, I will revert it .

File: mllib/src/test/scala/org/apache/spark/mllib/evaluation/MultilabelMetricsSuite.scala (hunk 3 above)
Comment (zhengruifeng) [inline mllib/src/test/scala/org/apache/spark/mllib/evaluation/MultilabelMetricsSuite.scala]: I change it for the IDEA complaint here

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**